### DISCUSSION TITLE
中国内部的敌对势力有五个来源

### DISCUSSION THREAD

**Discussion by lxn7022** [The-Run-Philosophy-Organization/run#436] [General]
1. 一个是中国革命的对象中的残留的部分。中国革命的对象是谁呢？帝国主义、封建主义和官僚资本主义。他们在中国的总代表就是美国反华分子扶持的国民党反动集团蒋介石政权。蒋介石政权虽然被赶到台湾去了，但他们中的绝大部分没有走，留在了大陆，有些是以土匪、特务的身份留了下来，有的就是普通的公务人员或其它社会人员。土匪、特务被清理掉了，其余的大部分接受了社会改造，变成了中国人民中的一员。还有少部分骨子里恨新生共和国的人民政权，欲推翻之而后快。这少部分人或他们的某些后代就成了新中国内部的敌对势力的一员。
2. 二是在历次运动中受到冤枉和打击的，其中大部分因被平反而重新回到人民的队伍里，但也有少部分人怀恨在心，也欲推翻新生的人民政权而后快。
3. 三是境外敌对势力通过金钱诱惑、舆论诱导、思想教唆等手段在中国内部培养的代理人，前面的第一部分和第二部分中都有相当的部分被境外敌对势力选中，变成潜藏在中国内部的敌对势力的一员。还有一部分是被金钱等物质利益诱惑的新生一代敌视中国体制的人员。
4. 四是台独、港独、藏独、疆独分子以及一切有类似企图的分子。
5. 五是接受过中国法律制裁的人中极少数怀恨在心底的分子。
[毒教材事件令人震惊，中国当警惕祸起萧墙之内](https://www.toutiao.com/article/7102803353993789956)
![image](https://user-images.githubusercontent.com/66630801/170905527-0dd3e808-c005-4093-9476-dfa4c38ff43a.png)
![image](https://user-images.githubusercontent.com/66630801/170905559-19e0cfa2-377b-4a40-99fe-ec04685bb17c.png)
![image](https://user-images.githubusercontent.com/66630801/170905677-9d8b731f-71e6-4b45-b25a-0f5ab1f04331.png)
![image](https://user-images.githubusercontent.com/66630801/170905707-cc4f5449-aa6d-4d0e-ba5d-96de90847b67.png)
![image](https://user-images.githubusercontent.com/66630801/170905747-12b6ab95-d59e-4240-a2b4-40009037ec93.png)

**Comment by AntEffect** [comment]


**Reply by lxn7022** [reply]
> 有没有一种可能，我是说可能哈，反对共产党政府跟反对中国是两回事？

江山就是人民，人民就是江山

**Reply by lxn7022** [reply]
两个确立， 两个维护，不管你喜欢还是不喜欢

**Comment by Evcklli** [comment]
干什么呢，润是为了更好的爱国
哈哈，那你把这些人全部杀了贵支不就是最强的的国家了？哈

**Comment by Evcklli** [comment]
包子：你要是真的反美别把女儿送美国啊，还改革开放干什么
